### PR TITLE
[Dictionary] Update setProp

### PR DESCRIPTION
**Pull request by livecodealice**
Put `Form:` on a new line and changed it to `**Form:**` for visibility.
Adjusted indentation of code examples and removed empty line from them.
Fixed broken links.

### CODE REVIEW

File: docs/dictionary/control_st/setProp.lcdoc [hunk 1]
```diff
@@ -60,24 +62,23 @@ itself.)
...
 > sets the <custom property> for a different <object(glossary)>, and the
-> first <object(glossary)> is in the second <object|object's> <message
-> path>, a runaway <recursion> will result. For example, if the
+> first <object(glossary)> is in the second <object|object's>
+> <message path>, a runaway <recursion> will result. For example, if the
 > following <handler> is in a <card> <script>, and you set the
 > "myCustomProperty" of a <button(object)> on the <card>, runaway
 > <recursion> will result:
 
     setProp myCustomProperty newValue
```
Comment (montegoulding) [inline docs/dictionary/control_st/setProp.lcdoc]: @livecodesam this should probably be changed so the param is `pValue`

File: docs/dictionary/control_st/setProp.lcdoc [hunk 2]
```diff
@@ -95,14 +96,13 @@ use array notation in the first line of the <setProp> <handler>, as in
 the following example:
 
     setProp mySet[thisProperty] newValue
```
Comment (montegoulding) [inline docs/dictionary/control_st/setProp.lcdoc]: @livecodesam again `pProperty` & `pValue`

File: docs/dictionary/control_st/setProp.lcdoc [hunk 3]
```diff
@@ -60,24 +62,23 @@ itself.)
...
 > <recursion> will result:
 
-    setProp myCustomProperty newValue
-    set the myCustomProperty of the target to newValue + 1
-    -- Because the target is the button, and this handler is in
-    -- the card, the above statement sends another setProp trigger
-    -- to the button.
-
+    setProp myCustomProperty pValue
+        set the myCustomProperty of the target to newValue + 1
```
Comment (livecodeali) [inline docs/dictionary/control_st/setProp.lcdoc]: this `newValue` needs to be `pValue` too

File: docs/dictionary/control_st/setProp.lcdoc [hunk 4]
```diff
@@ -95,8 +95,8 @@ If the custom property you want to control is in a custom property set,
 use array notation in the first line of the <setProp> <handler>, as in
 the following example:
 
-    setProp mySet[thisProperty] newValue
-        if thisProperty is "that" then put newValue into me
+    setProp mySet[thisProperty] pValue
+        if thisProperty is "that" then put pValue into me
```
Comment (livecodeali) [inline docs/dictionary/control_st/setProp.lcdoc]: These `thisProperty`s should probably be `pProperty`s

**Comment by montegoulding** [comment]
@livecode-vulcan review ok ffb930b

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok ffb930bbe9646ddf6e95c1e7bb725d8b58a1a497

**Comment by livecode-vulcan** [comment]
😎 test success ffb930bbe9646ddf6e95c1e7bb725d8b58a1a497

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/791)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/807)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/781)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/725)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/728)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/785)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/807)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/790)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/807)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/789)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/800)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/821)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/785)

**Pull request merged**